app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1114,6 +1114,18 @@ BrowserWorld::CreateTray() {
   m.tray->SetTransform(transform);
 }
 
+void
+BrowserWorld::SetTrayVisible(bool visible) const {
+  if (!m.tray)
```

Comment:
Tray already had a method to make it visible or invisible: tray->Toggle(bool aEnabled)

Reply:
Fixed